submissions/Zakkarat/html-css-popup/index.html

```diff
@@ -0,0 +1,107 @@
+<!DOCTYPE html>
+<html lang="en" dir="ltr">
+  <head>
+    <meta charset="utf-8">
+    <link rel="stylesheet" href="./css/main.css">
+    <link href="https://fonts.googleapis.com/css?family=Roboto" rel="stylesheet">
+    <title>Popup</title>
+  </head>
+  <body>
+        <div class="main">
+          <a href="#">LMail</a>
+          <a href="#">Images</a>
+          <input type="checkbox" id ="menu">
+          <label for="menu"><img src="./images/icons.png"></label>
```

Comment:
Don't forget fill `alt` attribute to the `img` tag.

Reply:
Done!